elkscmd/basic/host-8018x.c

```diff
@@ -89,3 +89,6 @@ int host_inpb(int port) {
 int host_inpw(int port) {
     return inw(port);
 }
+
+void host_exit() {
+}
```

Comment:
Remove all the `host_exit` definitions from all files except host-ibmpc.c. This is because the main basic.c never calls host_exit, it should instead be a static function in host-ibmpc.c only (and probably renamed to something like mode_reset so as to not be confused with the other host_xxx calls required by all host*.c handlers).